perfkitbenchmarker/kubernetes/kubernetes_virtual_machine.py

```diff
@@ -215,14 +205,11 @@ def _Exists(self):
     POD should have been already created but this is a double check.
     """
     exists_cmd = [FLAGS.kubectl, '--kubeconfig=%s' % FLAGS.kubeconfig, 'get',
-                  'pod', '-o=json', '-l', '%s=%s' % (SELECTOR_PREFIX,
-                                                     self.name)]
+                  'pod', '-o=json', self.name]
     pod_info, _, _ = vm_util.IssueCommand(exists_cmd, suppress_warning=True)
-    pod_info = json.loads(pod_info)
-
-    if len(pod_info['items']) == 0:
-      return False
-    return True
+    if pod_info:
```

Comment:
Yes. '-l' parameter is removed and now I will get no output instead of empty JSON document